recipes/spydrpick/build.sh

```
@@ -1,16 +1,32 @@
+git clone --branch v0.5.0 https://github.com/santeripuranen/apegrunt.git externals/apegrunt
+
+if [[ `uname` == "Darwin" ]]; then
+	export CONFIG_ARGS="-DCMAKE_FIND_FRAMEWORK=NEVER -DCMAKE_FIND_APPBUNDLE=NEVER"
+else
+	export CONFIG_ARGS=""
+fi
 
 # build and install SpydrPick (but only the SpydrPick make target)
-export CMAKE_MODULE_PATH=${RECIPE_DIR}
+export CMAKE_MODULE_PATH="${RECIPE_DIR}"
 mkdir build && pushd build
```

Shellcheck

[warning] Use 'pushd ... || exit' or 'pushd ... || return' in case pushd fails.

(SC2164)